src/Contrib/OtlpHttp/Exporter.php

```diff
@@ -51,17 +51,23 @@ public function __construct(
         ClientInterface $client,
         RequestFactoryInterface $requestFactory,
         StreamFactoryInterface $streamFactory,
-        SpanConverter $spanConverter = null
+        SpanConverter $spanConverter = null,
+        string $endpointUrl = null,
```

Comment:
If you use the default value from below here (localhost:4318), the if below is not required